.storybook/stories/TableFiltering/lib/testData.ts

```diff
@@ -0,0 +1,1498 @@
+		city: 'Port Rosalindaberg',
+		state: 'NSW',
+		status: 'completed',
+		requestDate: new Date('2022-07-18T14:48:41.119Z'),
+		numberOfEmployees: 717,
+	},
+	{
+		id: '0ab51c04-0f39-4671-ac3f-5f23c43b259e',
+		assignee: undefined,
+		businessName: 'Beahan, Grady and Ratke',
+		city: 'Fort Jeremyboro',
+		state: 'SA',
+		status: 'notBooked',
+		requestDate: new Date('2022-09-03T13:37:07.808Z'),
+		numberOfEmployees: 808,
+	},
+	{
+		id: '23aa5a15-72c0-4244-ac61-1e1d66ad803e',
+		assignee: 'Lando Norris',
+		businessName: 'Wehner and Sons',
+		city: 'Fort Lisandro',
+		state: 'WA',
+		status: 'cancelled',
+		requestDate: new Date('2022-07-16T05:30:52.866Z'),
+		numberOfEmployees: 808,
+	},
+	{
+		id: 'c23bd191-836c-48f9-8982-0525f95ab959',
+		assignee: 'Oscar Piastri',
+		businessName: 'Watsica Inc',
+		city: 'Padbergborough',
+		state: 'QLD',
+		status: 'booked',
+		requestDate: new Date('2022-03-19T18:45:21.166Z'),
+		numberOfEmployees: 965,
+	},
+	{
+		id: '056e5307-40c3-4f9e-9cac-f6c499416695',
+		assignee: 'George Russell',
+		businessName: 'Dicki, Trantow and Kuhic',
+		city: 'Cruickshankburgh',
+		state: 'VIC',
+		status: 'booked',
+		requestDate: new Date('2023-01-11T20:12:31.220Z'),
+		numberOfEmployees: 297,
+	},
+	{
+		id: 'd2bf7062-fae9-4e54-8870-447833b8822e',
+		assignee: undefined,
+		businessName: 'Barrows LLC',
+		city: 'East Elnora',
+		state: 'SA',
+		status: 'notBooked',
+		requestDate: new Date('2022-08-07T04:11:41.487Z'),
+		numberOfEmployees: 45,
+	},
+	{
+		id: '972d23c1-026e-4014-92d0-d4aeaa78eb28',
+		assignee: 'Lando Norris',
+		businessName: 'Gulgowski, Pfannerstill and Sipes',
+		city: 'Greenfort',
+		state: 'QLD',
+		status: 'notBooked',
+		requestDate: new Date('2022-07-15T11:53:46.397Z'),
+		numberOfEmployees: 685,
+	},
+	{
+		id: '77b4ac20-6bcc-4584-80e4-13ddf7aa8195',
+		assignee: 'George Russell',
+		businessName: 'Hahn, Moen and Champlin',
+		city: 'New Emmanuelle',
+		state: 'NSW',
+		status: 'completed',
+		requestDate: new Date('2022-07-27T23:53:50.951Z'),
+		numberOfEmployees: 245,
+	},
+	{
+		id: 'f993d3ea-80e9-407c-9db6-220659c2533c',
+		assignee: 'Lando Norris',
+		businessName: 'Bergnaum - Hessel',
+		city: 'North Damion',
+		state: 'TAS',
+		status: 'notBooked',
+		requestDate: new Date('2022-07-22T10:59:15.539Z'),
+		numberOfEmployees: 336,
+	},
+	{
+		id: 'ae17966e-6fab-40d1-8d1a-ef7bd27035fe',
+		assignee: 'Oscar Piastri',
+		businessName: 'Stoltenberg, Feest and Stehr',
+		city: 'Larryborough',
+		state: 'NSW',
+		status: 'booked',
+		requestDate: new Date('2023-03-03T00:32:39.110Z'),
+		numberOfEmployees: 73,
+	},
+];
+
+export function useTestData({ sort, filters, pagination }: GetDataParams) {
+	const { page, perPage } = pagination;
+
+	const filteredData = testData.filter((business) =>
+		filterData(business, filters)
+	);
+
+	const sortedData = filteredData
+		.sort((a, b) => sortData(a, b, sort))
+		.map((business, index) => ({
+			...business,
+			index: index + 1,
```

Comment:
```suggestion
			index,
```

I would like to keep 0-based indexing, so can we move this logic to where it is being used?